tests/system/python/packages/test_rule_data_availability.py

```diff
@@ -0,0 +1,278 @@
+# FLEDGE_BEGIN
+# See: http://fledge-iot.readthedocs.io/
+# FLEDGE_END
+
+""" Test data availability notification rule system tests:
+        Creates notification instance with data availability rule
+        and notify asset plugin for triggering the notifications based on CONCH.
+"""
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2023 Dianomic Systems, Inc."
+
+
+import os
+import subprocess
+import time
+import urllib.parse
+import json
+from pathlib import Path
+import http
+from datetime import datetime
+import pytest
+import utils
+from pytest import PKG_MGR
+
+# This  gives the path of directory where fledge is cloned. test_file < packages < python < system < tests < ROOT
+PROJECT_ROOT = Path(__file__).parent.parent.parent.parent.parent
```

Comment:
We can also use`PROJECT_ROOT = subprocess.getoutput("git rev-parse --show-toplevel")`